src/scripts/eastern_kingdoms/eastern_plaguelands/naxxramas/instance_naxxramas.cpp

```diff
@@ -307,40 +307,40 @@ void instance_naxxramas::UpdateTeleporters(uint32 uiType, uint32 uiData)
         case TYPE_MAEXXNA:
             if (GameObject* pGO = GetSingleGameObjectFromStorage(GO_ARAC_EYE_BOSS))
                 SetTeleporterVisualState(pGO, uiData);
-        
```

Comment:
there were spaces before the newline here before that git kept deleting automatically I think